coverage/clover.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<coverage generated="1678368858788" clover="3.2.0">
+  <project timestamp="1678368858788" name="All files">
+    <metrics statements="1" coveredstatements="1" conditionals="0" coveredconditionals="0" methods="1" coveredmethods="1" elements="2" coveredelements="2" complexity="0" loc="1" ncloc="1" packages="1" files="1" classes="1"/>
+    <file name="sum.js" path="C:\Users\Administrator\Desktop\atlp projects\destructors-ec-bn\src\sum.js">
+      <metrics statements="1" coveredstatements="1" conditionals="0" coveredconditionals="0" methods="1" coveredmethods="1"/>
+      <line num="2" count="1" type="stmt"/>
+    </file>
+  </project>
+</coverage>
```

Comment:
@divinecharlotte , remember to put coverage folder in **gitignore**

ex: coverage